docs/CHANGELOG.md

```diff
@@ -9,6 +9,124 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+#### New env vars
+
+`BLOCK_EMISSION_IDLE_WARNING_THRESHOLD` - controls global override for the time after which node will start logging warnings if no heads are received.
```

Comment:
I think we want newlines between each env var here o/w it renders like one paragraph.